xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamPVRManager.cpp

```diff
@@ -268,15 +268,20 @@ bool CDVDInputStreamPVRManager::PrevChannel(bool preview/* = false*/)
 bool CDVDInputStreamPVRManager::SelectChannelByNumber(unsigned int iChannelNumber)
 {
   PVR_CLIENT client;
+  CPVRChannelPtr currentChannel(g_PVRManager.GetCurrentChannel());
+  CFileItemPtr item(g_PVRChannelGroups->Get(currentChannel->IsRadio())->GetSelectedGroup()->GetByChannelNumber(iChannelNumber));
```

Comment:
you can leave it as it is, i only want to suggest to use the pvrmanager instead of the other pvr globals